api/src/services/cards.service.ts

```diff
@@ -12,25 +12,14 @@ import { ObjectId } from 'mongoose';
 @Service()
 export class CardService {
   public async findAllCards(page: number, limit: number, searchByName: boolean, cardName: string | undefined): Promise<PokemonCard[]> {
```

Comment:
I think you can delete the searchByName parameter here (and in the controller) since it's no longer used